client/src/App.js

```diff
@@ -22,7 +22,11 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Link to="/">MovieList</Link>
```

Comment:
I'm curious why you decided to put a Route path as well as a link here.

Reply:
Oh, must have left that in there by accident, from when I was toying with the links, thanks for the comments!